pkg/util/mount/mount_linux.go

```diff
@@ -1005,6 +1005,9 @@ func (mounter *Mounter) SafeMakeDir(subdir string, base string, perm os.FileMode
 }
 
 func (mounter *Mounter) GetMountRefs(pathname string) ([]string, error) {
+	if _, err := os.Stat(pathname); os.IsNotExist(err) {
```

Comment:
Add unit test?

I think the other GetMountRefs implementations also need to be updated.

Reply:
Updated windows and nsenter. They dont have unit tests for this so I didn't add anything. But I added a test case in `mount_linux_test.go`